config/application.rb

```diff
@@ -10,6 +10,7 @@
 require 'action_mailer/railtie'
 # require "action_view/railtie"
 # require "sprockets/railtie"
+require_relative '../lib/statsd_middleware'
```

Comment:
Any reason to prefer`require_relative` here vs `require`? I know that changes to application.rb mean you have to restart your rails server (even in development mode), but curious why you prefer require_relative here. Other than the fact that it is probably a little bit more performant than `require`